### ISSUE TITLE
Conda standalone JupyterLab env cannot detect `torch` from another kernel-env

### ISSUE THREAD

**Issue by nyngwang** [jupyter-lsp/jupyterlab-lsp#926]
## What I am trying to do...

TL;DR: Does your project `jupyterlab-lsp` assume that people will install both JupyterLab and PyTorch inside the same virtual environment? I created a standalone env. for JupyterLab and now your extension cannot detect PyTorch.

I have JupyterLab installed in a standalone conda env. named `jupyterlab` and PyTorch in another one called `pytorch-research`. Currently, I have installed `pyright` into `jupyterlab` since it will report that lsp is missing otherwise.

Now the problem is: The notebook cannot recognize the keyword `torch` since it's from the env `pytorch-research`.

This is my `jupyter_server_config.json`: (from [there](https://github.com/jupyter-lsp/jupyterlab-lsp/issues/534#issuecomment-782544899))

```
{
  "LanguageServerManager": {
    "language_servers": {
      "pyright-language": {
        "version": 2,
        "argv": ["pyright-langserver", "--stdio"],
        "languages": ["python"]
      }
    }
  }
}
```

## How I would like to learn how to do it...

What should I do so that I can have `pyright` installed and thus shared by all kernel-env(e.g. `pytorch-research` in my case) while I can keep dependencies outside of `jupyterlab`?

This is the current status:

<img src="https://user-images.githubusercontent.com/24765272/229930478-ef954556-f213-4568-bb7e-ee9a6bc1c8ff.png" width="600">
